### DISCUSSION TITLE
NSR: does CompCor also correct for global signal?

### DISCUSSION THREAD

**Discussion by Mihai1313** [FCP-INDI/C-PAC#1696] [Q&A]
Hi everyone,

I hope you can help me with one issue regarding NSR in C-PAC. It concerns global signal regression. As CSF and WM contain some variance related to GSR, it appears possible that CompCor may actually account for some correction similar to GSR. This would of course be related to how the CSF and WM look like, and it seems that the default pipeline includes erosion (thus minimizing a possible effect). But it's unclear to me what the level of erosion is (e.g., 30mm?). For instance, there is some evidence that eroding the WM twice and CSF four times removes the risk of unwilling correcting for global signal (Power et al., 2017: https://doi.org/10.1016/j.neuroimage.2016.09.038). I'm trying to avoid the possibility of regressing the global signal "twice", once via CompCor and once via GSR, as I have both active in my pipeline. Is there a risk of this? The CSF and WM masks used in the NSR also appear to have a high threshold (0.95), so I am uncertain whether additional erosion procedures (i.e., in addition to the default) are necessary. I am also uncertain how the default erosion (30mm) corresponds to masks twice or four times eroded, like mentioned in the Power paper referenced above.

I'd really appreciate your help!

Cheers,
Mihai

**Comment by tergeorge** [comment]
Hi Mihai, 

Sorry for the delay in getting back to you. In order to answer your question, we wanted to compare the intensity distribution of the 4 conditions - no regressors applied, applying both GSR and CompCorr, applying only GSR and applying only CompCorr. Then, we wanted to look at intensities in Gray matter, White matter, and CSF tissue of the preprocessed fMRI signal after running CPAC. 

Attached below are the intensity distribution plots of the three tissues for the various conditions:
![histogram](https://user-images.githubusercontent.com/88160056/161352637-48a2876e-59a3-4598-8ad4-fdf2a1978a75.png)
Visually, we can see that the differences between the GSR + CC setting, and the CC setting on its own, is subtle. Both signals generally have a similar dynamic range, and are zero centered, with a slightly longer tail in the only CC setting.


 While the signal distributions are not identical, the mean and variances of the GM signal intensity distributions are similar (0.007 ± 73.3 and 0.098 ± 80.21, for the GSR+CC and CC settings, respectively), suggesting that a "global offset" component may not be removed twice in the GSR+CC setting. Without simulated data or a test case it is impossible to say if a particular component is "double-corrected", but from these figures it is clear that both of these regressors perform unique corrections, and combining them may be of interest. In either case, it will be important to document the decision taken in your work.

Regards, 
Teresa
